source/nodejs/adaptivecards-extras-designer/src/adaptivecards-extras-designer.ts

```diff
@@ -0,0 +1,9 @@
+import { CardDesignerSurface } from "adaptivecards-designer";
```

Comment:
missing license/copyright comment (should be in every source file)

Reply:
Thanks, will fix in all affected files.